changelogs/fragments/71097-ansible-test-acme-container.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- "ansible-test - the ACME test container was updated, it now supports external account creation and has a basic OCSP responder (https://github.com/ansible/ansible/pull/71097, https://github.com/ansible/acme-test-container/releases/tag/2.0.0)."
```

Comment:
Include a newline at the end of the file. Other than that, looks good.

Reply:
Fixed in c451823d4fc4f9da42cda5e0cc54d03b2e111609.